keps/672-serial-job-execution/README.md

```diff
@@ -445,6 +525,7 @@ milestones with these graduation criteria:
 ## Implementation History
 
 - Draft KEP: September 25th 2024
+- Add Support for Multiple `DependsOn` Target: April 13th 2025
```

Comment:
@andreyvelich I think we should add a few more details here.

Maybe KEP merge and implementation?

Reply:
SGTM